test/OpenTelemetry.Instrumentation.AspNetCore.Tests/OpenTelemetry.Instrumentation.AspNetCore.Tests.csproj

```diff
@@ -27,7 +27,7 @@
     <Compile Include="$(RepoRoot)\test\OpenTelemetry.Tests\Shared\TestEventListener.cs" Link="Includes\TestEventListener.cs" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' == 'net7.0'">
+  <ItemGroup Condition="'$(TargetFramework)' == 'net7.0' or '$(TargetFramework)' == 'net8.0'">
```

Comment:
We should use the latest version here https://www.nuget.org/packages/Microsoft.AspNetCore.Mvc.Testing/8.0.0-rc.1.23421.29 for `.net8.0`

Reply:
Curious, what does this mean?

Reply:
Based on the compatible targetframework. https://www.nuget.org/packages/Microsoft.AspNetCore.Mvc.Testing/8.0.0-rc.1.23421.29#supportedframeworks-body-tab.

Ideally, we should try get rid of this dependency from our test by following https://github.com/open-telemetry/opentelemetry-dotnet/blob/8ea6bb951caa40b11c36ac68317a5a62eb32192d/test/OpenTelemetry.Instrumentation.AspNetCore.Tests/InProcServerTests.cs#L50

Reply:
I think we should take a different approach #4911.

Reply:
> Ideally, we should try get rid of this dependency from our test by following
> 
> https://github.com/open-telemetry/opentelemetry-dotnet/blob/8ea6bb951caa40b11c36ac68317a5a62eb32192d/test/OpenTelemetry.Instrumentation.AspNetCore.Tests/InProcServerTests.cs#L50

Reducing dependency would also be great 👍

Reply:
Fixed by #4911.